### PR TITLE
hides the second installation for MicroShift because there is no cont…

### PR DESCRIPTION
**Pull request by stevsmit**
PR does the following: 

Adds the IMPORTANT admonition
Swaps MicroShift -> {product-title}
Hides the other install guide, since we have no content for it yet

4.12 only. 

Preview: https://53971--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html

Pending SME approval. 

Jira: https://issues.redhat.com//browse/OSDOCS-4711

### CODE REVIEW

**Review by copejon — APPROVED** [review]
👍🏻

**Comment by Amrita42** [comment]
Please add the jira number in the description

File: microshift_install/microshift-install-rpm.adoc [hunk 1]
```diff
@@ -5,19 +5,22 @@ include::_attributes/attributes-microshift.adoc[]
 :context: microshift-install-rpm
 toc::[]
 
-You can install MicroShift from an RPM package on a machine with {op-system-first} {op-system-version}. 
+You can install {product-title} from an RPM package on a machine with {op-system-first} {op-system-version}. 
 
-To embed MicroShift in a {op-system-ostree} image, see xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Embedding MicroShift into a {op-system-ostree-first} image].
+[IMPORTANT]
+====
+Red Hat will not provide or support an update or upgrade path from Developer Preview and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```
Comment (Amrita42) [inline microshift_install/microshift-install-rpm.adoc:12]:
```suggestion
Red Hat will not provide or support an update or upgrade path from the Developer and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```

Comment (stevsmit) [reply microshift_install/microshift-install-rpm.adoc:12]: The original text is QE/SME approved so I don't think I'll change it.

**Pull request merged**